jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -344,42 +366,21 @@
       (setq obj-box-z-length (z-of-cube (send self :bbox->cube obj-box)))
       (setq obj-pos (v+ obj-pos (float-vector 0 0 (/ obj-box-z-length 2))))
       obj-pos))
-  (:try-to-pick-object
+  (:try-to-pick-object-v4
```

Comment:
`:try-to-pick-object-v4` is for current left hand.
This method is almost the same as `:try-to-pick-object` before #2155 
`:try-to-suction-object` and `:try-to-pinch-object` are for gripper-v6.